pipeline/beam_tables.py

```diff
@@ -597,7 +611,10 @@ def run_beam_pipeline(self, scan_type: str, incremental_load: bool,
 
       # TODO: Eventually we want to be able to run both exports simultaneously
       # so that we can reuse the pipeline output.
-      if export_gcs and gcs_folder is not None:
+      if bq_and_gcs:
+        self._write_to_bigquery(scan_type, rows, table_name, incremental_load)
```

Comment:
I would just leave this to-be-implemented for now and raise an exception. We can't test it until we have the outputs in sync.

Reply:
I implemented a simple function for that, but I can remove it again for now.